homeworkweek3/src/App.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from "react";
+import Header from "./components/header";
+import Laptops from "./components/laptops";
+
+class App extends Component {
+  state = {
+    showInfo: false
+  };
+  onShowClick = e => {
```

Comment:
Nu e gresit asa, insa dupa cum observi, ai semnatura cu un parametru pe care nu il folosesti.
Metoda poate fi simplificata, incat in lista de parametrii sa fie doar cei utilizati:

`onShowClick = () => { this.setState({ showInfo: !this.state.showInfo }) };`

In momentul in care vei folosi metoda pe buton, e ok sa initializezi acolo o noua functie ce va chema functia ta.

Exemplu: 

`          <button
            type="button"
            className="btn btn-secondary"
            onClick={() => this.onShowClick()}
          >`